cdap-cli/src/main/java/co/cask/cdap/cli/commandset/ProcedureCommands.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.cli.commandset;
+
+import co.cask.cdap.cli.command.CallProcedureCommand;
+import co.cask.common.cli.Command;
+import co.cask.common.cli.CommandSet;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+
+/**
+ * Procedure commands.
+ */
+public class ProcedureCommands extends CommandSet<Command> {
```

Comment:
Probably shouldn't create a ``CommandSet`` for procedures, given that its only 1 command (like the ``ExecuteQueryCommand``) and will likely be removed?